SPECS/kubevirt/kubevirt.spec

```diff
@@ -19,8 +19,8 @@
 %global debug_package %{nil}
 Summary:        Container native virtualization
 Name:           kubevirt
-Version:        0.58.0
-Release:        7%{?dist}
+Version:        0.59.0
```

Comment:
Would you please change the PR title to something that is changelog friendly.
Also, for an upgrade we would need an update to the signature file as well.
Please fill the PR checklist as well.